inc/rest-api/class-post-fields.php

```diff
@@ -466,6 +483,9 @@ private function remove_recursion( $fm ) {
 
 		if ( ! empty( $fm->children ) ) {
 			foreach ( $fm->children as $index => $child ) {
+				if ( ! $fm->serialize_data && $fm->add_to_prefix ) {
```

Comment:
Same here, are these properties sure to exist or could this cause an undefined index error?
Have you thought about using the null safe operator in PHP 8 https://php.watch/versions/8.0/null-safe-operator?